src/main/java/com/twilio/TwilioBearerTokenAuth.java

```diff
@@ -69,6 +69,7 @@ private static BearerTokenTwilioRestClient buildOAuthRestClient() {
 
         builder.region(TwilioBearerTokenAuth.region);
         builder.edge(TwilioBearerTokenAuth.edge);
+        builder.tokenManager(TwilioBearerTokenAuth.tokenManager);
```

Comment:
We should not be taking tokenManager from customer, SDK should be taking following inputs from customer **clientId**, **clientSecret**, **grantType** from customer.
Above are mandatory.
So there would be 2 constructors
1. **clientId**, **clientSecret**, **grantType**
2. **clientId**, **clientSecret**, **grantType** and other variables which are not mandatory.